doc/Security.xml

```diff
@@ -947,6 +960,61 @@ Added certificate-based client authentication</revremark>
         </orderedlist>
       </section>
     </section>
+    <section>
+      <title>JWT-based client authorization</title>
+      <para>Unlike with HTTP Digest and Ws-USernameToken authentication, this specification defines how to associate clients with the different User Levels as defined in the ONVIF Core Specification. Devices are not expected to have the user credentials stored in their memory, instead they will verify that the user has been correctly authenticated by an external service based on Open ID Connect and authorize accessing different functions, based on the user level presented in  the supplied JWT embedded in the binary security token.</para>
+
+      <para>The inhalt of the binary security token is the base64-encoded JWT according to [RFC 7519].</para>
+      <para>The JOSE header shall declare that the encoded object is a JWT, and the JWT is a JWS that is MACed using the HMAC SHA-256. Additionally, it must include the kid claim as defined by [RFC 7515], which indicates the ID associated to the key used to sign the JWT. If the aboslute path of the JWK Set URL differns from  https://iss/.well-known/jwks.json, the header may include an optional jku claim.</para>
+      <para>The JWT shall include the following standard claims:</para>
```

Comment:
Should we talk about alg claim in the header?

Reply:
As far as I know, the `typ` and `alg` claims in the JOSE header are mandatory. Nevertheless, we can list them explicitly.

Reply:
Yes they are mandatory I just thought we could list them for completeness. Maybe we can add an example too.

Reply:
Ok for adding them. The example is at the end of the document in the appendix.